tests/misc.sh

```diff
@@ -201,6 +201,26 @@ supported()
 			return 1
 		fi
 		;;
+	rename_ctime)
+		# POSIX does not require a file system to update a file's ctime
+		# when it gets renamed, but some file systems choose to do it
+		# anyway.
+		# https://pubs.opengroup.org/onlinepubs/9699919799/functions/rename.html
+		case "${fs}" in
+		EXT4)
```

Comment:
Should we check for EXT3 as well (I realize it's pretty aged at this point)?

Reply:
We could,  but ext3 is not listed as a supported file system in the README.  If we were going to add it here, then we should probably try to add support throughout.